README.md

```diff
@@ -38,13 +38,31 @@ An [Elastic Email][] adapter for the [Bamboo][] email app for Elixir.
 
 4.  Follow the Bamboo [Getting Started Guide][getting_started].
 
-5.  To use ElasticEmail's [postBack][] functionality you can place a value in
+5.  To use [ElasticEmail's additional api params][email_send] you can place a value in
     the `Email#private` parameter:
 
     ```elixir
-    Email.put_private(email, :elastic_custom_vars, %{post_back: "your-post-back-value"})
+    Email.put_private(email, :elastic_custom_vars, %{post_back: "your-post-back-value", pool_name: "your-pool-name"})
```

Comment:
I’d like to propose an API change here. Instead of `Email.put_private(email, :elastic_custom_vars, %{})`, let‘s call it `:elastic_send_options`.